### PR TITLE
Docs: run storages upadte

### PR DESCRIPTION
**Pull request by nguyeda1**

### CODE REVIEW

File: docs/api_v2/api_v2_reference.apib [hunk 1]
```diff
@@ -1995,7 +1995,70 @@ parameter. You can also filter runs by status ([available statuses](https://docs
             - count: 2 (number, required)
             - items (array[RunShort1,RunShort2], required)
 
-## Run object [/v2/actor-runs/{runId}{?token,waitForFinish}]
+## Run object [/v2/actor-runs/{runId}{?token,waitForFinish,status}]
+
+### Run storage endpoints
+
+This is not a single endpoint, but an entire group of endpoints that lets you to
+retrieve the last run or any of its default storages.
```
Comment (mtrunkat) [inline docs/api_v2/api_v2_reference.apib:2003]: not the "last run" but "run object"

File: docs/api_v2/api_v2_reference.apib [hunk 2]
```diff
@@ -1995,7 +1995,70 @@ parameter. You can also filter runs by status ([available statuses](https://docs
...
+All the endpoints require an authentication token.
```
Comment (mtrunkat) [inline docs/api_v2/api_v2_reference.apib:2004]: This is not true, it's authenticated by run ID.

File: docs/api_v2/api_v2_reference.apib [hunk 3]
```diff
@@ -1995,7 +1995,70 @@ parameter. You can also filter runs by status ([available statuses](https://docs
...
+
+The endpoints accept the same HTTP methods and query parameters as
+the respective storage endpoints.
+The base path that represents the last actor run object is:
+
+`/v2/actor-runs/{runId}{?token,status}`
+
+
+Using the `status` query parameter you can ensure to only get a run with a certain status
```
Comment (mtrunkat) [inline docs/api_v2/api_v2_reference.apib:2013]: Is status really there? I don't think that the filtration is needed in this case compared to the last run endpoints

**Review by mtrunkat — CHANGES_REQUESTED** [review]

File: docs/api_v2/api_v2_reference.apib (hunk 1 above)
Comment (m-murasovs) [reply docs/api_v2/api_v2_reference.apib:2003]: also 'that lets you to retrieve' => 'lets you retrieve'

File: docs/api_v2/api_v2_reference.apib [hunk 4]
```diff
@@ -1995,7 +1995,70 @@ parameter. You can also filter runs by status ([available statuses](https://docs
...
+
+* `/v2/actor-runs/{runId}/request-queue/requests{?token,status}` [Request collection](#reference/request-queues/request-collection)
+* `/v2/actor-runs/{runId}/request-queue/requests/{requestId}{?token,status}` [Request collection](#reference/request-queues/request)
+* `/v2/actor-runs/{runId}/request-queue/head{?token,status}` [Queue head](#reference/request-queues/queue-head)
+
+For example, to download data from a dataset of the last succeeded actor run in XML format,
+send HTTP GET request to the following URL:
+
+```
+https://api.apify.com/v2/actor-runs/{runId}/dataset/items?token={yourApiToken}&format=xml&status=SUCCEEDED
```
Comment (m-murasovs) [inline docs/api_v2/api_v2_reference.apib:2052]: Here, please remove the token param from the URL - we want to encourage users to pass their auth token in the `authorization` header

**Review by m-murasovs — CHANGES_REQUESTED** [review]

File: docs/api_v2/api_v2_reference.apib (hunk 4 above)
Comment (nguyeda1) [reply docs/api_v2/api_v2_reference.apib:2052]: Should I remove it altogehter from everywhere and from Parameters as well?

File: docs/api_v2/api_v2_reference.apib (hunk 1 above)
Comment (nguyeda1) [reply docs/api_v2/api_v2_reference.apib:2003]: `This is not a single endpoint, but an entire group of endpoints lets you retrieve the last run or any of its default storages.` Does not make sense or does it? It probably should be something like `
This is not a single endpoint, but an entire group of endpoints letting you retrive the last run or any of its default storages.`

File: docs/api_v2/api_v2_reference.apib [hunk 5]
```diff
@@ -1997,6 +1997,62 @@ parameter. You can also filter runs by status ([available statuses](https://docs
...
+https://api.apify.com/v2/actor-runs/{runId}/dataset/items?format=xml
+```
+
+In order to save new items to the dataset, send HTTP POST request with JSON payload to the same URL.
+
++ Parameters
+
+    + runId: `3KH8gEpp4d8uQSe8T` (string, required) - Run ID.
+    + token: `rWLaYmvZeK55uatRrZib4xbZs` (string, required) - API authentication token.
+
```
Comment (drobnikj) [inline docs/api_v2/api_v2_reference.apib:2055]: Log endpoint is missing

**Review by drobnikj — CHANGES_REQUESTED** [review]
Can you do it same way as last run endpoint to be able to see `Run storage endpoints` in the left menu

**Comment by drobnikj** [comment]
cc how it looks for last run endpoint
Snímek obrazovky 2021-06-07 v 19 49 47
how it looks for run storage endpoints
Snímek obrazovky 2021-06-07 v 19 50 12

**Review by mtrunkat — CHANGES_REQUESTED** [review]

File: docs/api_v2/api_v2_reference.apib (hunk 4 above)
Comment (m-murasovs) [reply docs/api_v2/api_v2_reference.apib]: I'd say leave it in the params, and just remove from the sample URL, pls :)

**Review by mtrunkat — CHANGES_REQUESTED** [review]
I meant if the token is necessary in all the endpoints here - my guess is that all of them should work without it not only the `/v2/actor-runs/{runId}/log`

**Comment by nguyeda1** [comment]
> I meant if the token is necessary in all the endpoints here - my guess is that all of them should work without it not only the `/v2/actor-runs/{runId}/log`

Actually I think we do check for the token... in `actor_jobs.js => getActorRun` which is called by each endpoint.. I think it is the same for the last run

**Review by m-murasovs — APPROVED** [review]

**Review by mtrunkat — APPROVED** [review]

**Review by drobnikj — CHANGES_REQUESTED** [review]
Looks good, just last thing, I just change the order of text and put the Get Run endpoint at the beginning of the caption and after the storages. I feel it should be the first as it is more important than storages.

**Review by drobnikj — APPROVED** [review]

**Pull request merged**